helpers/request.ts

```diff
@@ -236,7 +239,7 @@ export function getCorsJSON(url: string, data?: RequestData): Promise<any> {
     .setData(data)
     .submit()
     .then((response) => {
-      if (response.status >= 200 && response.status < 300) {
+      if (response.status >= HTTP_STATUS.OK && response.status < HTTP_STATUS.MULTIPLE_CHOICES) {
```

Comment:
I think it because harder to understand like this, maybe we should still rely on 200 and 300 magic numbers introduce a helper function `isSuccessStatus` that those this check to not have them in too many places ?